src/main/java/Duke.java

```diff
@@ -1,10 +1,62 @@
+import java.util.Scanner;
+import java.util.ArrayList;
 public class Duke {
+    public static void printHorizontalLine() {
```

Comment:
Method names can be changed to PascalCase

Reply:
I believe that names representing methods must be verbs and written in camelCase.